test/builder/builder_test.cc

```diff
@@ -275,6 +276,49 @@ TEST_F(RoadGeometryBuilderTest, LaneGeometryBuilderWithMissingLineStrings) {
       maliput::common::assertion_error);
 }
 
+TEST_F(RoadGeometryBuilderTest, LaneGeometryBuilderWithAndWithoutCenter) {
+  // No throws as LaneGeometryBuilder is properly used.
+  // Centerline isn't passed to the builder.
```

Comment:
Nit: move this up, outside the test body to keep the style.

Reply:
Done.